vtex/faststore
# Feat: `ProductShelf` Section Override API v2

## What's the purpose of this pull request?

This PR adapts `ProductShelf` to use Section Override API v2.

## How to test it?

See the instructions in the starter.store PR.

- https://github.com/vtex-sites/starter.store/pull/320

## References

- https://github.com/vtex/faststore/pull/2091
- https://github.com/vtex-sites/starter.store/pull/246
